scripts/cli/src/commands/util/init.ts

```diff
@@ -0,0 +1,131 @@
+import { AccountId, Address } from "@polkadot/types/interfaces/runtime";
+import { Type } from "@polkadot/types";
+import { SubmittableExtrinsic } from '@polkadot/api/types';
+import { IKeyringPair , ISubmittableResult } from '@polkadot/types/types';
+import { cryptoWaitReady, blake2AsHex, mnemonicGenerate } from "@polkadot/util-crypto";
+import { ApiPromise, WsProvider, Keyring } from "@polkadot/api";
+import BN from "bn.js";
+import fs from "fs";
+import path from "path";
+import { KeyringPair } from '@polkadot/keyring/types';
+
+export let api: ApiPromise;
+
+export type Account = {
+    address: Address,
+    isLocked: Boolean,
+    meta: {name: String},
+    publicKey: AccountId,
+    type: Type
+}
+
+export let nonces = new Map();
+
+export async function setAPI(endpoint: string) {
+
+   // Schema path
+   const filePath = path.join(__dirname + "/../../../../../polymesh_schema.json");
+   const { types } = JSON.parse(fs.readFileSync(filePath, "utf8"));
+ 
+   // Start node instance
+   const ws_provider = new WsProvider(endpoint);
+    api = await ApiPromise.create({
+     types,
+     provider: ws_provider
+   });
+
+}
+
+export function getAPI() {
+  return api;
+}
+
+export async function generateEntity(api: ApiPromise, name: string) {
```

Comment:
Probably it would be nice to be able to generate an entity this way, or by providing a private key instead (or a different mneomic, rather than just a derivation path).

Reply:
good point, I could probably have some type parameter that depending on the type given, the function would generate the entity based off of that.

Reply:
I will add this as a separate pull request since its a little bigger than the suggestions.